### PR TITLE
refactor(backend): eslint plugin import + fixes

### PR DESCRIPTION
**Pull request by ulfgebhardt**
### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/src/password/EncryptorUtils.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { User } from '@entity/User'
 import CONFIG from '@/config'
 import LogError from '@/server/LogError'
 import { backendLogger as logger } from '@/server/logger'
-import { User } from '@entity/User'
 import { PasswordEncryptionType } from '@enum/PasswordEncryptionType'
 
-// eslint-disable-next-line @typescript-eslint/no-var-requires
+// eslint-disable-next-line @typescript-eslint/no-var-requires, import/no-commonjs
```
Comment (Mogge) [inline backend/src/password/EncryptorUtils.ts:10]: Can this be done by an import?

File: backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { User as DbUser } from '@entity/User'
 import { Event as DbEvent } from '@entity/Event'
+/* eslint-disable-next-line import/no-cycle */
```
Comment (Mogge) [inline backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts:3]: disable this rule?

File: backend/src/password/EncryptorUtils.ts (hunk 1 above)
Comment (ulfgebhardt) [reply backend/src/password/EncryptorUtils.ts:10]: I'd like to investigate this in another PR - there was a problem with `import { ... } from ' sodium-native' ` - thats why its required.
I made an Issue: https://github.com/gradido/gradido/issues/2903

File: backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts (hunk 2 above)
Comment (ulfgebhardt) [reply backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts]: Solution I implemented now: disable rule

**Review by Elweyn — COMMENTED** [review]
Let's define which rules we find good and which one we dislike.

File: backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts (hunk 2 above)
Comment (ulfgebhardt) [reply backend/src/event/EVENT_ACTIVATE_ACCOUNT.ts]: Better solution proposed in https://github.com/gradido/gradido/pull/2905

**Review by Mogge — APPROVED** [review]

**Pull request merged**